app/Console/Commands/FindWarnings.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
should we keep this? it is pretty after all :D

Reply:
I was on the fence.  I'll keep it for now and delete later, maybe. :)